### DISCUSSION TITLE
Possible shit

### DISCUSSION THREAD

**Discussion by Bunjolio** [3kh0/ext-remover#654] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My school give us play store. but it only has certain apps we can download.. is there possibly a way to exploit this to give me apps that usually are undownloadable?

**Comment by 3kh0** [comment, marked as answer]
no

**Reply by Bunjolio** [reply]
i gotchu chief

**Comment by RavenChill-Dev** [comment]
If you guys have Linux enabled on your chromebooks, then sure, that would work, and just download the .apk files you want.

**Reply by Bunjolio** [reply]
well sadly i dont.. i hate this retarded ahh school on bro

**Reply by RavenChill-Dev** [reply]
lol
